entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+CLOUD_PROVIDER=${1:-$CLOUD_PROVIDER}
+CRI_RUNTIME_ENDPOINT=${CRI_RUNTIME_ENDPOINT:-/run/peerpod/cri-runtime.sock}
```

Comment:
please change the default to `/run/containerd/containerd.sock`

Reply:
note that this is the socket path which is exposed in the containers, the host's socket path default is `/run/containerd/containerd.sock`

Reply:
I see.. thanks for the clarification